.readthedocs.yaml

```diff
@@ -1,11 +1,18 @@
+# Configuration on how ReadTheDocs (RTD) builds our documentation
+# ref: https://readthedocs.org/projects/2i2c-pilot-hubs/
+# ref: https://docs.readthedocs.io/en/stable/config-file/v2.html
+#
 version: 2
 
-conda:
-  environment: docs/environment.yml
-
 build:
-  image: latest
+  os: ubuntu-20.04
+  tools:
+    # NOTE: Update the python version hand in hand with the requirement
+    #       specification in docs/environment.yml
+    python: "3.8"
```

Comment:
```suggestion
    # NOTE: This must be mambaforge or it will mistakenly try to invoke `python` to do a conda install
    # ref: https://stackoverflow.com/questions/69774313/why-does-rtd-run-python-env-create-instead-of-conda-env-create-when-building
    python: "mambaforge-4.10"

```